bundles/prometheus.0.15.0.substitutesfor/manifests/servicemonitor.crd.yaml

```diff
@@ -0,0 +1,224 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
nit: same note about CRD versioning ie using `v1` instead of `v1beta1`. I can see that since real CRDs and operators are being used in the tests this can be annoying, but its a case of doing this now or in a few months